RM-1376: ProxySQL 2.6.2 Release

venkatesh-prasad-v opened the pull request:
https://perconadev.atlassian.net/browse/RM-1376

***
Fix the unstable bats test.

***
This patch also introduces two new options `--no-kill-mysqld` and `--no-kill-proxysql` to proxysql-admin-testsuite.sh to skip killing of mysqld and proxysql processes before the tests are run.

Commit:
RM-1376: ProxySQL 2.6.2 Release

https://perconadev.atlassian.net/browse/RM-1376

***
Fix the unstable bats test.

***
This patch also introduces two new options `--no-kill-mysqld` and
`--no-kill-proxysql` to proxysql-admin-testsuite.sh to skip killing of
mysqld and proxysql processes before the tests are run.

kaushikpuneet07 reviewed (approved):
Patch tested and it works good

Commit:
PSQLADM-511: proxysql-admin syncusers hangs at 1000 DB user or above

https://perconadev.atlassian.net/browse/PSQLADM-511

Follow-up fix:
Passwords having single quote (') characters were not processed
properly due to the missing of escaping.

kaushikpuneet07 reviewed (approved):
LGTM
